hal/src/main/native/include/hal/HALBase.h

```diff
@@ -77,14 +77,16 @@ int64_t HAL_GetFPGARevision(int32_t* status);
 /**
  * Returns the serial number.
  *
- * @return Serial number.
```

Comment:
Unless the returned `size_t` is a success indicator, there should be a `@return` tag

Reply:
the returned `size_t` indicates how many characters were copied into `buffer`. I'll go ahead and add one.